src/mlpack/methods/kernel_svm/kernel_svm.hpp

```diff
@@ -0,0 +1,257 @@
+/**
+ * @file methods/kernel_svm/kernel_svm.hpp
+ * @author Himanshu Pathak
+ *
+ * An implementation of Kernel SVM.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_KERNEL_SVM_KERNEL_SVM_HPP
+#define MLPACK_METHODS_KERNEL_SVM_KERNEL_SVM_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <ensmallen.hpp>
+
+
+namespace mlpack {
+namespace svm {
+
+/**
+ * The KernelSVM class.
+ *
+ *
+ * @tparam MatType Type of data matrix.
+ */
+template <typename MatType = arma::mat, typename KernelType = >
```

Comment:
I case you are wondering why this doesn't build, this is missing a kernelType.

Reply:
Yup, changed it also I haven't added CMake file. Now, all builds will fail because of my rough implementation :smile: